invideoquiz/invideoquiz.py

```diff
@@ -48,7 +48,7 @@ class InVideoQuizXBlock(StudioEditableXBlockMixin, XBlock):
 
     timemap = String(
         display_name=_('Problem Timemap'),
-        default={},
+        default='',
```

Comment:
Because this is a string field, it it actually just displays as the string "None"